internal/services/notification/db/migrate.go

```diff
@@ -69,3 +70,41 @@ func (d *DB) migrateV3(tx *sql.Tx) error {
 
 	return nil
 }
+
+func (d *DB) migrateV4(tx *sql.Tx) error {
+	var dmlGeneric = "DELETE FROM runwebhookdelivery"
```

Comment:
so the migration will delete all the current webhook data?

Reply:
@sgotti I can leave webhook data but it will not have webhook deliveries.

Reply:
Can you be more detailed?

Since the notification service changes are done in this development cycle could just drop all the notification service data. But if we follow this line we could just also remove all the notifications service migrations until the official next release (people using master should manually clean the notification service db).

Instead if we're doing migrations then we should correctly migrate.

Reply:
ok @sgotti we can do migration, I can get ProjectID from Payload and populate the new column using a temporary table.